src/lib/draw/InterventionLayer.svelte

```diff
@@ -70,6 +72,20 @@
       colors.other,
       "white",
     ];
+  // For planning mode
+  const colorByReferenceType: DataDrivenPropertyValueSpecification<string> = [
```

Comment:
There's boilerplate here and above; we can refactor if we do this kind of thing often